docs/outline.toml

```diff
@@ -164,7 +164,7 @@ classes = ["PrefectResult", "GCSResult", "LocalResult", "S3Result", "AzureResult
 [pages.engine.serializers]
 title = "Result Serializers"
 module = "prefect.engine.serializers"
-classes = ["Serializer","JSONSerializer"]
+classes = ["PickleSerializer","JSONSerializer"]
```

Comment:
When the base class `Serializer` is added, it should also be added to the docs here.